detectable/src/main/java/com/synopsys/integration/detectable/factory/DetectableFactory.java

```diff
@@ -279,7 +282,7 @@ public DetectableFactory(FileFinder fileFinder, DetectableExecutableRunner execu
     //Should only be accessed through the DetectableFactory.
 
     public DockerDetectable createDockerDetectable(DetectableEnvironment environment, DockerDetectableOptions dockerDetectableOptions, DockerInspectorResolver dockerInspectorResolver, JavaResolver javaResolver,
-        DockerResolver dockerResolver) {
+                                                   DockerResolver dockerResolver) {
```

Comment:
It appears your formatter is not up to date due to the indentation changes. Also this line doesn't seem long enough to need wrapping.